must-gather/collection-scripts/gather

```diff
@@ -65,6 +66,9 @@ resources+=(rolebindings)
 resources+=(configmaps)
 resources+=(serviceaccounts)
 resources+=(events)
+# clusterlogging is only in the openshift-logging namespace, but it should not impact a lot to run the inspect in the other namespaces
```

Comment:
This statement is not correct.  With the introduction of multi CLF, users can add a CL resource to change resources and schedulting of their forwarder